File: Python/day8.py
phone_book = {}
for i in range(int(raw_input())):
    name = raw_input()
    phone = int(raw_input())
    phone_book[name] = phone

while True:
    try:
        name = raw_input()
        if name in phone_book:
            print "=".join([name, str(phone_book[name])])
        else:
            print "Not found"
    except EOFError:
        break

File: Ruby/day8.rb
n = gets.to_i
phone_book = {}
(1..n).each do |i|
    name = gets.chomp
    phone = gets.to_i
    phone_book[name] = phone
end

queries = $stdin.read
queries.split("\n").each do |name|
    if phone_book.has_key?(name)
        puts [name, phone_book[name]].join("=")
    else
        puts "Not found"
    end
end
